custom_navbutton_completion.js

```diff
@@ -0,0 +1,119 @@
+/**
```

Comment:
Not sure on this, tried to check on a quick glance and seems like some work doing so maybe could take this up as a further enhancement in later versions?